src/args/lint-with-ajv/ajv-cli/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
     "ajv-cli": "*"
+  },
+  "overrides": {
```

Comment:
@rohaquinlop Did you test the builtin?

Reply:
Yes

Reply:
![image](https://github.com/fluidattacks/makes/assets/50106623/6a40d8f1-ce22-4a98-9c5c-802f08f71fab)